src/sentry/api/endpoints/organization_events_meta.py

```diff
@@ -39,6 +41,10 @@ def get(self, request: Request, organization) -> Response:
                 params=params,
                 query=request.query_params.get("query"),
                 referrer=Referrer.API_ORGANIZATION_EVENTS_META.value,
+                # TODO: @athena - add query_source when all datasets support it
+                # query_source=(
+                #     QuerySource.FRONTEND if is_frontend_request(request) else QuerySource.API
+                # ),
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, not all datasets currently expect query_source as a param and they're not typed to safely take care of that. So I'll uncomment it after my last PR.